packages/scalecube-microservice/src/Microservices/Microservices.ts

```diff
@@ -1,16 +1,17 @@
+import uuidv4 from 'uuid/v4';
 import createDiscovery from '@scalecube/scalecube-discovery';
 import { defaultRouter } from '../Routers/default';
 import { getProxy } from '../Proxy/Proxy';
 import { getServiceCall } from '../ServiceCall/ServiceCall';
-import { uuidv4 } from '../helpers/utils';
+import { isNodeEnv } from '../helpers/utils'
 import { createServiceRegistry } from '../Registry/ServiceRegistry';
 import { createMethodRegistry } from '../Registry/MethodRegistry';
 import { MicroserviceContext } from '../helpers/types';
 import { Endpoint, Message, Microservice, MicroserviceOptions, Microservices as MicroservicesInterface } from '../api';
 import { ASYNC_MODEL_TYPES, MICROSERVICE_NOT_EXISTS } from '../helpers/constants';
 
 export const Microservices: MicroservicesInterface = Object.freeze({
-  create: ({ services, seedAddress = location.hostname }: MicroserviceOptions): Microservice => {
+  create: ({ services, seedAddress = isNodeEnv() ? 'clusterAddress' : location.hostname }: MicroserviceOptions): Microservice => {
```

Comment:
location.hostname, seem pointless in here, "defaultSeedAddress" will work much better